keyboards/lily58/lib/glcdfont_lily.c

```diff
@@ -1,6 +1,11 @@
 // This is the 'classic' fixed-space bitmap font for Adafruit_GFX since 1.0.
 // See gfxfont.h for newer custom bitmap font info.
 
+// Modified to show the Lily58 logo instead of the qmk logo
+
+#ifndef FONT5X7_H
```

Comment:
this was removed on the master, now in this branch this `#ifndef` has no `#endif`